meta-resin-common/recipes-connectivity/networkmanager/networkmanager_1.10.2.bb

```diff
@@ -8,7 +8,7 @@ SECTION = "net/misc"
 LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://COPYING;md5=cbbffd568227ada506640fe950a4823b \
                     file://libnm-util/COPYING;md5=1c4fa765d6eb3cd2fbd84344a1b816cd \
-                    file://docs/api/html/license.html;md5=eb0c003714e5391000fdfd9c9615cccf \
+                    file://docs/api/html/license.html;md5=697fa2cd28065955314b0bcf5ab1ff04 \
```

Comment:
What changed in the license? I'm asking just to make sure we don't have a license change.

Reply:
--- license.old 2017-11-10 16:48:27.000000000 +0100
+++ license.new 2017-12-12 02:52:14.000000000 +0100
@@ -7,7 +7,7 @@
 <link rel="home" href="index.html" title="NetworkManager Reference Manual">
 <link rel="up" href="index.html" title="NetworkManager Reference Manual">
 <link rel="prev" href="ix01.html" title="Index">
-<meta name="generator" content="GTK-Doc V1.25 (XML mode)">
+<meta name="generator" content="GTK-Doc V1.26 (XML mode)">
 <link rel="stylesheet" href="style.css" type="text/css">
 </head>
 <body bgcolor="white" text="black" link="#0000FF" vlink="#840084" alink="#0000FF">
@@ -379,6 +379,6 @@
     </p>
 </div>
 <div class="footer">
-<hr>Generated by GTK-Doc V1.25</div>
+<hr>Generated by GTK-Doc V1.26</div>
 </body>
 </html>
\ No newline at end of file

Reply:
-<meta name="generator" content="GTK-Doc V1.25 (XML mode)">
+<meta name="generator" content="GTK-Doc V1.26 (XML mode)">

Reply:
Ok. Looks good. It would be nice to mention this in the commit log too. It looks like it is due to gtk doc version change.